src/lib/bedrockYaml.test.ts

```diff
@@ -1,4 +1,4 @@
-import uuid from "uuid/v4";
+import uuid = require("uuid/v4");
```

Comment:
the `uuid` library uses a really outdated method for exporting its functions:

```ts
import { v4 } from './interfaces';

declare const v4: v4;

export = v4
```

the `export = v4` is non-standard and only worked with our old `import...from` call because our tsconfig has `"esModuleInterop": true`. However, since we've added `"exclude": ["**/*.test.ts"]` to our tsconfig as well, a lot of refactoring tools and linters flag ` import uuid from "uuid/v4";` as incorrect in tests files.